src/main/java/com/epam/tc/api/hw3/utils/TestData.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.tc.api.hw3.utils;
+
+public class TestData {
```

Comment:
This name is a bit confusing as there is also TestData class with data providers and this class is in utils. Also it looks like BASE_URL can be moved to EndPoints or CommonService and KEY, TOKEN can be moved to CommonService as they are used there.